### PR TITLE
Update Ko-fi

### LINKED ISSUE #11971: Update: Ko-fi
### Brand Name

Ko-fi

### Official Resources for Icon and Color

New Icon: https://ko-fi.com/post/Ko-fis-updated-look-K3K714GIG2
https://storage.ko-fi.com/cdn/brandasset/v2/kofi_symbol.png

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by ameknite**
kofi(1)

**Issue:** closes #11971

**Popularity metric:**
https://www.similarweb.com/website/ko-fi.com/: 6,310

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Ko-fi new logo: https://ko-fi.com/post/Ko-fis-updated-look-K3K714GIG2
It was converted from the png using inkscape.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8495,9 +8495,9 @@
         },
         {
             "title": "Ko-fi",
-            "hex": "FF5E5B",
-            "source": "https://more.ko-fi.com/brand-assets",
-            "guidelines": "https://more.ko-fi.com/brand-assets"
+            "hex": "FC5A17",
```
Comment (uncenter) [inline _data/simple-icons.json:8498]:
```suggestion
            "hex": "FF5E5B",
```

Color is still `#FF5E5B` according to Ko-fi's branding page.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8495,9 +8495,9 @@
...
+            "source": "https://storage.ko-fi.com/cdn/brandasset/v2/kofi_symbol.png",
+            "guidelines": "https://help.ko-fi.com/hc/en-us/articles/360007937553-Ko-fi-Content-Guidelines"
```
Comment (uncenter) [inline _data/simple-icons.json:8500]: Let's convert the EPS assets from the branding page so we have a more accurate SVG than a traced one.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (ameknite) [reply _data/simple-icons.json]: It is #FF6433
Screenshot 2024-10-08 at 21 15 39

Comment (ameknite) [reply _data/simple-icons.json]: https://ko-fi.com/post/October-Challenge-Rediscover-your-creative-joy-O4O2146NKM

File: _data/simple-icons.json (hunk 2 above)
Comment (ameknite) [reply _data/simple-icons.json:8500]: It was converted using the trace bitmap function of inksacape.
What is EPS? do you have a tool for that?

File: _data/simple-icons.json (hunk 1 above)
Comment (uncenter) [reply _data/simple-icons.json]: That isn't their branding guidelines though. It says this on the brand page:

image

Comment (uncenter) [reply _data/simple-icons.json]: Ah,  my apologies ahah. Looks like they haven't even updated the logos there! Maybe we should reach out...

**Comment by uncenter** [comment]
I've reached out to them to hopefully see if they can update their branding page so we can use that.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -8495,9 +8495,9 @@
         },
         {
             "title": "Ko-fi",
-            "hex": "FF5E5B",
-            "source": "https://more.ko-fi.com/brand-assets",
-            "guidelines": "https://more.ko-fi.com/brand-assets"
+            "hex": "FF6433",
+            "source": "https://storage.ko-fi.com/cdn/brandasset/v2/kofi_symbol.png",
+            "guidelines": "https://help.ko-fi.com/hc/en-us/articles/360007937553-Ko-fi-Content-Guidelines"
```
Comment (uncenter) [inline _data/simple-icons.json:8500]: Can we revert these to their original links?

File: icons/kofi.svg
Comment (uncenter) [inline icons/kofi.svg:1]: Seeing some differences between their provided SVG (`kofi_symbol.svg` from the brand assets download) and your traced version:

CleanShot 2024-11-09 at 19 31 57

**Review by uncenter — CHANGES_REQUESTED** [review]
Sweet! Looks like https://more.ko-fi.com/brand-assets was updated.

**Comment by uncenter** [comment]
@ameknite are you still interested in working on this pull request?

**Comment by ameknite** [comment]
I'm busy, I hope someone else can work on it.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @ameknite and @uncenter!

**Pull request merged**